index.html

```diff
@@ -0,0 +1,175 @@
+        <p class="message">
+          A fun festival where people from more than 80 countries who believe in
+          the value of sharing and openness and create positive change gather
+          and share ideas will be held in Korea in October.
+        </p>
+        <div class="location">
+          <h1 class="date">2022.10.27 (Fri) ~ 29 (Sun)</h1>
+          <h5 class="locate">
+            @ Islamabad club pakistan.
+          </h5>
+        </div>
+      </section>
+    </div>
+
+    <!-- Program Section start -->
+
+    <section class="program" id="programs">
+      <div class="main_heading">
+        <h1 class="heading">Main Program</h1>
+        <div class="guidbar"></div>
+      </div>
+      <div class="programListWrpper">
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_01.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">Lecture</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_02.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">CC Exhibition</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_03.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">Forum</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_04.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">workshop</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+        <div class="program_list">
+          <div class="program_icon">
+            <img src="img/program_icon_05.png" alt="Image"/>
+          </div>
+          <h5 class="program_title">CC Ignite</h5>
+          <p class="program_description">
+            Listen to lectures from speakers from around the world and learn
+            about the latest trends in the world.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="join" id="join">
+        Join PE Summit 2022
+      </button>
+      <a href="#programs" class="see">SEE THE WHOLE PROGRAM</a>
+    </section>
```

Comment:
- [ ] The font size for the text that has been used in this section is smaller than the one used in the Behance design. I am aware that increasing the font size might distort the design, for this I would recommend reducing the number of words to keep the design in check.

**This is how I am seeing it:**

![image](https://user-images.githubusercontent.com/86318284/198354788-969e9bdb-6185-43a2-bed8-4765646e77b5.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198354872-1413c343-edbe-452b-88d2-ac4f6924d7c8.png)